README.md

```diff
@@ -1,3 +1,17 @@
+ASP.NET Core IISIntegration
+========
 This repo hosts the ASP.NET Core middleware for IIS integration and the ASP.NET Core Module.
 
 This project is part of ASP.NET Core. You can find samples, documentation and getting started instructions for ASP.NET Core at the [Home](https://github.com/aspnet/home) repo.
+
+## Building from source
+1. Install prerequisites
+   1. Visual Studio 2017
+      1. Workload: `Desktop development with C++`
+         1. Optional: `Windows 8.1 SDK and UCRT SDK`
```

Comment:
Optional is confusing as it makes it seem like it is an optional install, rather than a required optional component.

Reply:
Renamed `Optional` to `Additional Component`